crates/ty_python_semantic/resources/mdtest/scopes/global.md

```diff
@@ -83,7 +83,7 @@ def f():
     x = 1
     def g() -> None:
         nonlocal x
-        global x  # TODO: error: [invalid-syntax] "name 'x' is nonlocal and global"
+        global x  # error: [invalid-syntax] "name 'x' is nonlocal and global"
```

Comment:
This `error:` assert should start passing when I fill in some other TODOs below.